sql/telemetry_derived/firefox_nondesktop_day_2_7_activation_v1/query.sql

```diff
@@ -34,12 +12,13 @@ WITH base AS (
       AND BIT_COUNT(days_seen_bits << 1 & udf.bitmask_lowest_7()) > 0
     ) AS day_2_7_activated,
   FROM
-    `moz-fx-data-shared-prod.telemetry.nondesktop_clients_last_seen_v1`
+    `moz-fx-data-shared-prod.telemetry.nondesktop_clients_last_seen`
+  WHERE
+    contributes_to_2020_kpi
```

Comment:
I like the addition of this where clause. Took a look a the [dashboard](https://datastudio.google.com/reporting/1L7dsFyqjT8XZHrYprYS-HCP5_k_gZGIb/page/0iERB) for the day 2-7 activation and it looks like it's currently only including products that contribute to the 2020 KPI only. I initially thought that was happening in the dashboard but it isn't. Looked at the query for creating the day_2_7 table and the clients_last_seen table and I'm not seeing where the exclusion is happening. The day 2-7 just miraculously knows what our KPI products are for 2020. Am I missing something?

Reply:
Later on in this query there's an existing clause:

```
  AND product IN (
    'Fenix',
    'Firefox Preview',
    'Fennec Android',
    'Focus Android',
    'Fennec iOS',
    'Focus iOS',
    'Firefox Lite',
    'FirefoxConnect',
    'Lockwise Android'
  )
```

That's what's causing the filtering you see today. I'll need to remove that clause as part of this PR, since this `contributes_to_2020_kpi` line handles the same intention.